### ISSUE TITLE
Broken?

### ISSUE THREAD

**Issue by willt** [eska014/aseprite-import#6]
I don't get any errors on import but the scene it creates doesn't  contain any animations. 
Exporting as Horizontal Strip, All Layers, All Frames Output and Json checked saved to same folder,
Hash selected Layers, Frame Tags, Slices all checked.

**Comment by willt** [comment]
Looks like in aseprite you have to select the frames right click and add frame tag. This is the name aseprite-import uses for the animation name. I was expecting it to use the layer name.

**Comment by leonkrause** [comment]
Right, the frame tags are used as animations. I should make this more obvious in the readme file. Thanks for the feedback

**Comment by orphu** [comment]
Perhaps an error if no frame tags are found? I struggled for a bit with this as well.
